src/main/java/tc/oc/pgm/commands/ModerationCommands.java

```diff
@@ -18,16 +24,40 @@
 
 public class ModerationCommands {
 
+  private static final int REPORT_COOLDOWN_SECONDS = 15;
+
+  private final Map<CommandSender, Instant> lastReportSent = new WeakHashMap<>();
```

Comment:
Should use a guava expiring cache, and maybe just use the UUID as key (if the command sender is the console just always allow)

Reply:
Also exempt players with the `Permissions.STAFF`

Reply:
Thanks @Pablete1234 and @Electroid! 
I switched the  map over to use a guava loading cache. I haven’t used them before, so hopefully the way I implemented it is the proper way. Though after testing everything seemed to work, so it should be right 😄 

Also I added the staff bypass.